drivers/platform/mpam/mpam_devices.c

```diff
@@ -1696,6 +1696,33 @@ static void mpam_pcc_rx_callback(struct mbox_client *cl, void *msg)
 	/* TODO: wake up tasks blocked on this MSC's PCC channel */
 }
 
+static const struct mpam_quirk mpam_quirks[] = {
```

Comment:
This commit looks more from MPAM-dev than from James 6.10-rc1. Same for the next 4 commits about workaround seems taken from MPAM-dev than from James 6.10-rc1.

Reply:
because the nvidia workaround of JM_6.10-rc1 introduces the new data structures,
I refer the version of MPAM-dev and have the cleaner backport.